CodeEdit/NavigatorSidebar/FindNavigator/FindNavigator.swift

```diff
@@ -16,6 +16,10 @@ struct FindNavigator: View {
     @State
     private var searchText: String = ""
 
+    @State var filters: [String] = ["Ignoring Case", "Matching Case"]
```

Comment:
Could we replace this array with an enum? Using an array like this feels breakable, especially with only two possible values.

Reply:
Yeah I turned it into a string enum